pandas/core/series.py

```diff
@@ -4977,11 +4977,14 @@ def _cmp_method(self, other, op):
 
         if isinstance(other, Series) and not self._indexed_same(other):
             raise ValueError("Can only compare identically-labeled Series objects")
+        if isinstance(other, MultiIndex):
```

Comment:
here is the another usage with MI as we discussed in another PR. ok to unify this later, just pointing this out. that MI are an odd man out from the rest of the array-likes we have .

Reply:
agreed